requirements.txt

```diff
@@ -1,69 +1,56 @@
-alabaster==0.7.10
 alembic==0.9.9
+asn1crypto==0.24.0
 Babel==2.6.0
 bcrypt==3.1.4
 beautifulsoup4==4.6.0
-cffi>=1.7.0
+certifi==2019.3.9
+cffi==1.7.0
+chardet==3.0.4
+Click==7.0
 configobj==5.0.6
-cookies==2.2.1
-cov-core==1.15.0
-coverage==4.5.1
 croniter==0.3.24
 cryptography==2.4.2
 defusedxml==0.5.0
-docutils==0.14
 ecdsa==0.13
-enum34==1.1.6
 Flask==1.0.2
 Flask-Babel==0.11.2
 Flask-Migrate==2.1.1
 Flask-Script==2.0.6
 Flask-SQLAlchemy==2.3.2
-funcparserlib==0.3.6
-gunicorn==19.8.1
+httplib2==0.12.1
 huey==1.11.0
 idna==2.6
-imagesize==1.0.0
-ipaddress==1.0.22
-itsdangerous==0.24
+itsdangerous==1.1.0
 Jinja2==2.10
 ldap3==2.6
-lxml==4.2.1
+lxml==4.2.5
 Mako==1.0.7
-MarkupSafe==1.0
-mock==2.0.0
+MarkupSafe==1.1.1
 netaddr==0.7.19
-nose==1.3.7
-numpy==1.14.3
-oauth2client>=2.0.1
+oauth2client==2.0.1
 passlib==1.7.1
 Pillow==5.1.0
-psycopg2>=2.6
```

Comment:
I would like to have psycopg2 in the ubuntu packages.
To my understanding this would have to be compiled if installed manually in a virtualenv?
So we obviously should patch this into ubuntu packages but not have this in the requirements.txt?